src/domain/entities/testing/mocks.ts

```diff
@@ -31,20 +31,41 @@ export const MOCK_FIGMA_NODE_IDENTITY = new FigmaDesignIdentity(
 );
 export const MOCK_FIGMA_DESIGN_IDENTITY = MOCK_FIGMA_FILE_IDENTITY;
 
+function getRandomInt(min: number, max: number): number {
+	min = Math.ceil(min);
+	max = Math.floor(max);
+	return Math.floor(Math.random() * (max - min) + min);
+}
+
+export const generateFigmaFileName = () => uuidv4();
+
+export const generateFigmaFileKey = () =>
+	Buffer.from(uuidv4()).toString('base64');
+
+export const generateFigmaNodeId = () =>
+	`${getRandomInt(1, 100)}:${getRandomInt(1, 100)}`;
+
 export const generateFigmaDesignUrl = ({
 	fileKey = MOCK_FIGMA_FILE_KEY,
```

Comment:
I think this should probably be using `generateFigmaFileKey` instead of `MOCK_FIGMA_FILE_KEY`.

Reply:
Yes, great catch! 

✅ Fixed.